jira_integration/jira_integration.php

```diff
@@ -0,0 +1,111 @@
+<?php
+// TODO: this header should not be required...
+header('Content-Type: text/plain; charset=UTF-8');
+
+// Only watch the dev environment
+if ($_ENV['PANTHEON_ENVIRONMENT'] != 'dev') {
+  die();
+}
+
+// Retrieve jira destination and credentials
+$jira = json_decode(file_get_contents($_SERVER['HOME'] . '/files/private/jira_integration.json'), 1);
+if ($jira == FALSE) {
+  die('No jira file found. Aborting!');
+}
+
+$commits = array(
+  // The latest git hash
+  'current' => NULL,
+  // The hash at the last time we processed this script
+  'last_processed' => NULL,
+  // Raw output of git log since the last processed
+  'history_raw' => NULL,
+  // Formatted array of commits being sent to jira
+  'history' => array(),
+  // An array keyed by jira ticket id, each holding an
+  // array of commit ids.
+  'jira' => array()
+);
+
+// Get latest commit
+$commits['current'] = shell_exec('git rev-parse HEAD');
+
+// Retrieve the last commit processed by this script
+$commit_file = $_SERVER['HOME'] . '/files/private/jira_integration.commit';
```

Comment:
Change to the following:

`$commit_file = $_SERVER['HOME'] . '/files/private/'. $_ENV['PANTHEON_ENVIRONMENT'] .'_jira_integration.commit';`

Otherwise we could cross the streams between environments.